basic/atm/atm.py

```diff
@@ -0,0 +1,96 @@
+
+def delete_old_file(pfile):
+    """delete the old file of ATM data
+    """
+    pfile.seek(0)
+    pfile.truncate()
+    pfile.seek(0) # I believe this seek is redundant
+    return pfile
+def file_to_list(file):
+    """convert file to list
+        """
+    text = file.read()
+    list_1 = text.split('\n')
+    i = 0
+    list_2 = []
+    while i < len(list_1):
+        list_2 = list_2 + list_1[i].split(',')
+        i = i + 1
+    return list_2
+def dictionary_id_password(list):
+    """create dictionary id:password from list
+                """
+    i = 0
+    password_dct = {list[i]: list[i + 1] for i in range(0, len(list) - 1, 3)}
```

Comment:
I've understood what is happening only after another read, give names for some of the numbers which are constants.